### PR TITLE
Update Storybook to use SCSS files

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
This PR updates the settings and version of Storybook for us to be able to use it with `.scss` modules.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -1,52 +1,37 @@
...
+        cssLoaderOptions: {
+          modules: {
+            auto: true
+          }
+        }
+      }
+    },
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-a11y"
```
Comment (tifa2UP) [inline .storybook/main.js:19]: Did you remove this part on purpose?

```
 '@storybook/addon-storysource',
 '@storybook/addon-a11y/register',
```

File: .storybook/main.js [hunk 2]
```diff
@@ -1,52 +1,37 @@
...
   ],
   typescript: {
-    reactDocgen: false,
+    reactDocgen: false
+  },
+  "core": {
+    "builder": "webpack5"
   },
   webpackFinal: async (config) => {
```
Comment (tifa2UP) [inline .storybook/main.js:27]: good riddance, didn't like custom webpack setups in the first place. 

Thank you!

**Review by tifa2UP — APPROVED** [review]
Code looks good mA. Waiting for CI to pass

File: .storybook/main.js (hunk 1 above)
Comment (osamasayed) [reply .storybook/main.js:19]: `@storybook/addon-a11y` was set-up in `preview.js` which is not the [right way](https://www.npmjs.com/package/@storybook/addon-a11y) to set it up along with using `@storybook/addon-a11y/register`, so removed the usage from `preview.js` and added the correct syntax here.

**Pull request merged**